test/helpers/utils.go

```diff
@@ -577,6 +577,19 @@ func DoesNotRunWithKubeProxyReplacement() bool {
 	return !RunsWithKubeProxyReplacement()
 }
 
+// RunsWithKubeProxyReplacementOutsideGKE returns true if the kernel supports
+// our kube-proxy replacement and the test are running outside GKE. Note that
+// kube-proxy may still be running alongside Cilium.
+func RunsWithKubeProxyReplacementOutsideGKE() bool {
+	return RunsWithKubeProxyReplacement() && DoesNotRunOnGKE()
+}
```

Comment:
We're only using this helper function once. I'd rather we use an anonymous function as we do elsewhere.

Reply:
Sure! Done